enterprise/app/services/captain/copilot/chat_service.rb

```diff
@@ -27,10 +27,15 @@ def generate_response(input)
   def system_message
     {
       role: 'system',
-      content: Captain::Llm::SystemPromptsService.copilot_response_generator(@assistant.config['product_name'])
+      content: Captain::Llm::SystemPromptsService.copilot_response_generator(@assistant.config['product_name'], account_language)
```

Comment:
Can this be an input to the service and pass it from the controller? We could pass the user preferences in future.